index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css" />
+</head>
+<body>
+    <div class="header_view__container"> 
+        <span>Gmail</span>
+        <span>이미지</span>
+        <span>Apps</span>
+        <span>더보기</span>
+    </div>
```

Comment:
```suggestion
    <ul class="header_view__container">
        <li><a>Gmail</a></li>
        <li><a>이미지</a></li>
        <li><a>Apps</a></li>
        <li><a>더보기</a></li>
    </ul>
```

이렇게 하면 좀 더 구조적으로 될 것 같아요.